pkgs/data/icons/gruvbox-material-icons-gtk/default.nix

```
@@ -0,0 +1,43 @@
+{ lib,
+  stdenv,
+  fetchFromGitHub,
+  gtk3,
+  gtk-engine-murrine,
+  hicolor-icon-theme,
+  breeze-icons,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gruvbox-material-gtk";
+  version = "latest-2021-07-25";
+
+  src = fetchFromGitHub {
+    owner = "TheGreatMcPain";
+    repo = pname;
+    rev= "cc255d43322ad646bb35924accb0778d5e959626";
+    sha256="sha256-1O34p9iZelqRFBloOSZkxV0Z/7Jffciptpj3fwXPHbc=";
+  };
+
+  nativeBuildInputs = [ gtk3 ];
```

I'm guessing you meant this.
```suggestion
  buildInputs = [ gtk3 ];
```

---

This is better as gtk3 is only being used at build-time: gtk-update-icon-cache

---

I see. Alright.